HISTORY.md

```diff
@@ -1,5 +1,183 @@
+* `tabular` -> `single_table`
+* `relational` -> `multi_table`
+* `timeseries` -> `sequential`
+* `BaseTabularModel` -> `BaseSingleTableSynthesizer`
+* `GaussianCopula` -> `GaussianCopulaSynthesizer`
+* `CTGAN` -> `CTGANSynthesizer`
+* `TVAE` -> `TVAESynthesizer`
+* `CopulaGan` -> `CopulaGANSynthesizer`
+* `PAR` -> `PARSynthesizer`
+* `HMA1` -> `HMASynthesizer`
+
+In `SDV` 1.0, synthesizers are classes that take in metadata and handle data preprocessing, model training and model sampling. This is similar to the previous `BaseTabularModel` in `SDV` <1.0.
+
+`Synthesizers` in `SDV` 1.0 define a clear workflow for generating synthetic data.
+1. Synthesizers are initialized with a metadata class.
+2. They can then be used to transform the data and apply constraints using the `synthesizer.preprocess()` method. This step also validate that the data matches the provided metadata to avoid errors in fitting or sampling.
+3. The processed data can then be fed into the underlying machine learning model using `synthesizer.fit_processed_data()`. (Alternatively, data can be preprocessed and fit to the model using `synthesizer.fit()`.)
+4. Data can then be sampled using `synthesizer.sample()`.
+
+Each synthesizer class also provides a series of methods to help users customize the transformations their data goes through. Read more about that [here](https://docs.sdv.dev/sdv/single-table-data/modeling/synthetic-data-workflow/transform-and-anonymize).
+
+Notice that the preprocessing and model fitting steps can now be separated. This can be helpful if preprocessing is time consuming or if the data has been processed externally.
+
+Another major addition is control over randomization. In `SDV` <1.0, users could set a seed to control the randomization for only some columns. In `SDV` 1.0, randomization is controlled for all columns. Every new call to sample generates new data, but the synthesizers seed can be reset to the original state using `synthesizer.reset_randomization()`, enabling reproducibility.
+
+PII handling is improved by the following features:
+* Primary keys can be set to natural sdtypes (eg. SSN, email, name). Previously they could only be numerical or text.
+* The `PseudoAnonymizedFaker` can be used to provide consistent mapping to PII columns. The mapping itself can be accessed by viewing the transformers for the column using `synthesizer.get_transformers()`.
+* A bug causing PII columns to slow down modeling is patched.
+
+Finally, the synthetic data can now be easily evaluated using the `evaluate_quality()` and `run_diagnostic()` methods. The data can be compared visually to the actual data using the `get_column_plot()` and `get_column_pair_plot()` methods. For more info on how to visualize or interpret the synthetic data evaluation, read the docs [here](https://docs.sdv.dev/sdv/single-table-data/evaluation).
+
```

Comment:
Then maybe we can make this a new major section

```
## Details
```

Everything you have underneath it is then sub-headers of it.